core/modules/views/includes/view.inc

```diff
@@ -304,6 +304,7 @@ class view {
       // Convert all stored displays into display objects.
       foreach ($this->display as $display_id => $display) {
         $this->display[$display_id] = new views_display($display['display_plugin'], $display_id, $display['display_title'], $display['display_options']);
+        $this->display[$display_id]->position = 0;
```

Comment:
Rather than manually specifying the position each time a `views_display` is created, could we just specify `position` as a variable in the `views_display` class? e.g.
```
  /**
   * The position of this display relative to other displays.
   *
   * @var int
   */
  var $position = 0;
```